app/javascript/components/mentoring/discussion/EditDiscussionPost.tsx

```diff
@@ -12,12 +12,20 @@ export const EditDiscussionPost = ({
   endpoint: string
   contextId: string
 }): JSX.Element => {
-  const { open, setOpen, buttonRef, panelRef, styles, attributes } = usePanel()
+  const {
+    open,
+    setOpen,
+    componentRef,
+    buttonRef,
+    panelRef,
+    styles,
+    attributes,
+  } = usePanel()
 
   const handleSuccess = useCallback(() => setOpen(false), [setOpen])
 
   return (
-    <React.Fragment>
+    <div ref={componentRef}>
```

Comment:
Does this need to be a div? The CSS will break if there is this extra layer. Could we add this ref onto the button itself?

Reply:
It should be a div because we need to know when we click outside it to close the panel